suite-native/module-accounts-management/src/components/AccountDetailScreenHeader.tsx

```diff
@@ -43,6 +46,9 @@ export const AccountDetailScreenHeader = ({
                     onPress={handleSettingsNavigation}
                 />
             }
+            leftIcon={
+                <GoBackIcon closeActionType={closeActionType === 'close' ? 'close' : 'back'} />
```

Comment:
```suggestion
                <GoBackIcon closeActionType={closeActionType} />
```

Reply:
Or maybe just 
```suggestion
                <GoBackIcon closeActionType='close' />
```? 
As one of following comments suggests.

Reply:
ffd098bc78